Jenkinsfile.integration

```diff
@@ -199,7 +200,7 @@ pipeline {
         }
         cleanup {
             script {
-                    sh './run.sh teardown'
+                    sh '[ -f /tmp/${SOCOK8S_ENVNAME}.needcleanup ] && ./run.sh teardown && rm /tmp/${SOCOK8S_ENVNAME}.needcleanup'
```

Comment:
I think would be better reorder the teardown and the rm commands in case teardown exits with error and to be sure that the file is removed. Is interesting to create a new patch for that?

Reply:
not a bad idea.